doc/api/cli.md

```diff
@@ -288,6 +288,17 @@ Path to the file used to store the persistent REPL history. The default path is
 to an empty string (`""` or `" "`) disables persistent REPL history.
 
 
+### `NODE_TTY_UNSAFE_ASYNC=1`
+<!-- YAML
+added: REPLACEME
+-->
+
+When set to `1` writes to stdout and stderr will be non-blocking and async when
```

Comment:
Nit: comma after `1`

Nit: backticks around `stdout` and `stderr` 

Nit: `async`->`asynchronous`